backend/main.go

```diff
@@ -32,9 +32,13 @@ func main() {
 	}()
 	log.Printf("marking application liveness as UP")
 	health.ApplicationStatus.MarkAsUp()
+
 	log.Printf("Server started")
-	log.Printf("Authentication method: %s", singleton.GetAuthenticationMethod())
+	log.Printf("Authentication method: %s", clusterSingleton.GetAuthenticationMethod())
+
 	router := sw.NewRouter()
+	health.ServiceStatus.MarkAsUp()
+	log.Printf("marking application readiness as UP")
```

Comment:
This is duplicating the code below I think these two lines should be deleted

Reply:
Yeah, merge problem. Will adress tomorow.